pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -58,10 +58,6 @@ const (
 	// desiredStateOfWorldPopulatorLoopSleepPeriod is the amount of time the
 	// DesiredStateOfWorldPopulator loop waits between successive executions
 	desiredStateOfWorldPopulatorLoopSleepPeriod time.Duration = 1 * time.Minute
-
-	// reconcilerSyncDuration is the amount of time the reconciler sync states loop
-	// wait between successive executions
-	reconcilerSyncDuration time.Duration = 5 * time.Second
```

Comment:
The wrong volume will be only mounted if:

1. Volume is detached from pod, outside of k8s system (like AWS console)
2. Node is rebooted. 

#1 is not a huge concern because - usually we don't recommend users to detach volumes attached to nodes/pods. 

#2 is bit of a problem and I am damned for saying this - but as long as time it takes for a reboot to complete is more than time specified here - we would be okay.